eyefloaters/strimzi-ui
# Add basic integration with console-api backend

**What this PR does / why we need it**:

Integrate TS backend with [console-api](https://github.com/eyefloaters/console-api)

**Which issue(s) this PR fixes**

N/A

**Verification steps**

1. Deploy and run the console-api as described in the [README](https://github.com/eyefloaters/console-api/blob/65e4412d799b393034e495f9a6e725f93a07828e/README.md) (linked version is not yet on `main` branch).
2. Add `BACKEND_URL="http://127.0.0.1:8080"` to your `.env` file
3. Run `npm run dev`
4. Navigate to `/clusters` path in the UI

![image](https://github.com/eyefloaters/strimzi-ui/assets/20868526/fbe3306c-e422-4661-8278-898c53797e5d)

![image](https://github.com/eyefloaters/strimzi-ui/assets/20868526/5ec34bcc-c3e5-474d-b882-b6758b921685)


**Is there any work left / what's next** :

This is only a very rough example of integration with the console-api and is not expected to be merged as-is.

**Special notes for your reviewer**:
